website/blog/2020-11-10-lithium.md

```diff
@@ -0,0 +1,502 @@
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs,
+Sublime Text, Atom and Eclipse. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+- Organize imports code action.
+- Show implicit parameters and inferred type decorations.
+- Improved sbt BSP support.
+- Support for Scala 3.0.0-M1.
+- Remote debugging.
+- Environment variables in run/debug.
+
+## Organize imports code action
+
+One of the most requested features in the Metals repository, current list can be
+found [here](https://github.com/scalameta/metals/issues/707), was the ability to
+automatically organize imports. We are happy to announce, that thanks to
+[mlachkar's](https://github.com/mlachkar) amazing contributions to both
+[Scalafix](https://github.com/scalacenter/scalafix) and Metals, this new feature
+is now available via `Organize imports` code action.
+
+![imports](https://i.imgur.com/8YBdjjC.gif)
+
+Depending on the editor this code action can be invoked differently, please
+consult specific editor's documentation. For example in Visual Studio Code
+'organize imports' can be invoked from command console, shortcut, or from the
+dropdown menu when right clicking inside a file.
+
+The organize imports code action is enabled using
+[Scalafix](https://github.com/scalacenter/scalafix) and specifically the awesome
+organize imports rule created by [liancheng](https://github.com/liancheng).
+
+The rule can be used to automatically sort imports in a file alphabetically,
+which is the default setting, or use the user specific configuration defined
+using scalafix configuration file. This file can be either `.scalafix.conf` in
+the current workspace or an absolute file specified in
+`metals.scalafixConfigPath` user setting. It's important to note that the new
+code action is consistent with how sbt's scalafix plugin will behave.
+
+An example scalafix configuration for organize imports rule can look like this:
+
+```scala
+OrganizeImports {
+  groups = ["javax?\\.", "scala.", "*"]
+  removeUnused = true
+}
+```
+
+This will sort imports into 3 groups defined with regexes and remove any unused
+ones. Specifically, it will turn:
+
+```scala
+import scala.collection.mutable.{Buffer, ArrayBuffer}
+import java.time.Clock
+import java.lang.{Long => JLong, Double => JDouble}
+
+object RemoveUnused {
+  val buffer: ArrayBuffer[Int] = ArrayBuffer.empty[Int]
+  val long: JLong = JLong.parseLong("0")
+}
+```
+
+into
+
+```scala
+import java.lang.{Long => JLong}
+
+import scala.collection.mutable.ArrayBuffer
+
+object RemoveUnused {
+  val buffer: ArrayBuffer[Int] = ArrayBuffer.empty[Int]
+  val long: JLong = JLong.parseLong("0")
+}
+```
+
+More information can be found in the
```

Comment:
Is it maybe worth explicitly mentioning this warning:

> Please do NOT use the Scalafix built-in RemoveUnused.imports together with OrganizeImports to remove unused imports.

Reply:
Good idea, added!